mobile/package.json

```diff
@@ -41,4 +42,4 @@
     "eslint-plugin-react-hooks": "^4.6.0"
   },
   "private": true
-}
\ No newline at end of file
+}
```

Comment:
did you have to manually downgrade libraries? I'm a little confused why I see that the versions decreased in package.json